docs/reference/capabilities.md

```diff
@@ -84,6 +84,8 @@ about capabilities, refer to the [Appium documentation](https://appium.io/docs/e
 |`appium:shouldTerminateApp`| Specify if the app should be terminated on session end. This capability only has an effect if an application identifier has been passed to the test session (either explicitly, by setting bundleId, or implicitly, by providing app). Default is `true` unless `noReset` capability is set to `true`. |`true` or `false`|
 |`appium:forceAppLaunch`| Specify if the app should be forcefully restarted if it is already running on session startup. This capability only has an effect if an application identifier has been passed to the test session (either explicitly, by setting bundleId, or implicitly, by providing app). Default is `true` unless `noReset` capability is set to `true`. |`true` or `false`|
 |`appium:useNativeCachingStrategy`| Set this capability to `false` in order to use the custom elements caching strategy. This might help to avoid stale element exception on property change. By default the native XCTest cache resolution is used (`true`) for all native locators (e.g. all, but xpath). Check the corresponding [WebDriverAgent pull request](https://github.com/appium/WebDriverAgent/pull/516) for more details. |`true` or `false`|
+|`appium:skipDeviceCheck`| Whether to skip the device check in a new session request. This could help to avoid [Apple TV 4k tvOS 17](https://github.com/appium/appium/issues/19343) connection issue, but if you need this capability, the session will have communication issue with the device. Commands that can work over WebDriverAgent may work, but commands such as application installation may not work. (We do not recommend using this capability. Please use this only when you need this.) Default is `false`. | `true` or `false` |
```

Comment:
Maybe also mention this capability is temporary and we'll remove it as soon as we have a decent replacement to the ios-device lib

Reply:
or maybe we could just use devicectl to list devices and do other stuff, so then we'd have an option to not skip, but use devicectl instead of ios-device

Reply:
for example we could run `xcrun devicectl list devices` to list devices